aten/src/ATen/native/mkldnn/xpu/detail/oneDNNContext.h

```diff
@@ -18,6 +18,11 @@ TORCH_XPU_API dnnl::memory make_onednn_memory(
     dnnl::engine& engine,
     void* ptr);
 
+TORCH_XPU_API dnnl::memory make_onednn_memory(
+    dnnl::memory::desc md,
+    dnnl::engine& engine,
+    const void* ptr);
+
```

Comment:
Why does this function need to exposed?

Reply:
To support COW, the tensor data ptr should be set as const.
The make_onednn_memory() has no such instance.
So, add make_onednn_memory() with const ptr.